src/App.js

```diff
@@ -1,11 +1,12 @@
-import {PluralSightLoginClone} from "./projects";
+import {PluralSightLoginClone, ResponsivePhotographyWebsite} from "./projects";
 
 function App() {
   return (
       <>
-          <PluralSightLoginClone/>
+          <ResponsivePhotographyWebsite/>
+          {/*<PluralSightLoginClone/>*/}
```

Comment:
Why is it commented?

Reply:
will fix with routing system

Reply:
again, if you want to merge a commented piece of code I expect to see a todo with id of a task in the scope of which it will be handled

Reply:
@TaHUoP fixed